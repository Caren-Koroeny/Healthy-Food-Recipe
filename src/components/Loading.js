import React from 'react';
import '../styles/Meal.css';

function Loading() {
  return (
    <div className="loading">
      <h1>loading...</h1>
    </div>
  );
}

export default Loading;
