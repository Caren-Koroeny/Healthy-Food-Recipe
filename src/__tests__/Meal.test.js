import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import Loading from '../components/Loading';

describe('Loading Component', () => {
  it('Renders Loading', () => {
    const { container } = render(<Loading />);
    expect(screen.getByText('loading...')).toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot();
  });

  it('Renders only loading', () => {
    const { container } = render(<Loading />);
    expect(screen.queryByText('Welcome')).not.toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot();
  });
});
