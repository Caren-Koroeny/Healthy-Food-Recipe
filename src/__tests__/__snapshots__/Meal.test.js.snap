// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component Renders Loading 1`] = `
<div
  class="loading"
>
  <h1>
    loading...
  </h1>
</div>
`;

exports[`Loading Component Renders only loading 1`] = `
<div
  class="loading"
>
  <h1>
    loading...
  </h1>
</div>
`;
